content/en/docs/contribute/_index.md

```diff
@@ -28,41 +28,62 @@ Kubernetes documentation welcomes improvements from all contributors, new and ex
 
 ## Getting started
 
-Anyone can open an issue about documentation, or contribute a change with a pull request (PR) to the [`kubernetes/website` GitHub repository](https://github.com/kubernetes/website). You need to be comfortable with [git](https://git-scm.com/) and [GitHub](https://lab.github.com/) to operate effectively in the Kubernetes community.
+Anyone can open an issue about documentation, or contribute a change with a
+pull request (PR) to the
+[`kubernetes/website` GitHub repository](https://github.com/kubernetes/website).
+You need to be comfortable with
+[git](https://git-scm.com/) and
+[GitHub](https://lab.github.com/)
+to operate effectively in the Kubernetes community.
 
 To get involved with documentation:
 
 1. Sign the CNCF [Contributor License Agreement](https://github.com/kubernetes/community/blob/master/CLA.md).
-2. Familiarize yourself with the [documentation repository](https://github.com/kubernetes/website) and the website's [static site generator](https://gohugo.io).
-3. Make sure you understand the basic processes for [opening a pull request](/docs/contribute/new-content/new-content/) and [reviewing changes](/docs/contribute/review/reviewing-prs/).
+1. Familiarize yourself with the [documentation repository](https://github.com/kubernetes/website)
+   and the website's [static site generator](https://gohugo.io).
+1. Make sure you understand the basic processes for
+   [opening a pull request](/docs/contribute/new-content/open-a-pr/) and
+   [reviewing changes](/docs/contribute/review/reviewing-prs/).
 
 Some tasks require more trust and more access in the Kubernetes organization.
-See [Participating in SIG Docs](/docs/contribute/participating/) for more details about
+See [Participating in SIG Docs](/docs/contribute/participate/) for more details about
 roles and permissions.
 
 ## Your first contribution
 
-- Read the [Contribution overview](/docs/contribute/new-content/overview/) to learn about the different ways you can contribute.
-- See [Contribute to kubernetes/website](https://github.com/kubernetes/website/contribute) to find issues that make good entry points.
-- [Open a pull request using GitHub](/docs/contribute/new-content/new-content/#changes-using-github) to existing documentation and learn more about filing issues in GitHub.
-- [Review pull requests](/docs/contribute/review/reviewing-prs/) from other Kubernetes community members for accuracy and language.
-- Read the Kubernetes [content](/docs/contribute/style/content-guide/) and [style guides](/docs/contribute/style/style-guide/) so you can leave informed comments.
-- Learn about [page content types](/docs/contribute/style/page-content-types/) and [Hugo shortcodes](/docs/contribute/style/hugo-shortcodes/).
+- Read the [Contribution overview](/docs/contribute/new-content/overview/) to
+  learn about the different ways you can contribute.
+- See [Contribute to kubernetes/website](https://github.com/kubernetes/website/contribute)
```

Comment:
Should this be `/docs/contribute/` or another link?

Reply:
makes sense. Since the current link target is empty, and we have other links for generic contribution guidelines, I'm taking the liberty to revise it to a pointer to issues list:

```
- Check [kubernetes/website issues list](/https://github.com/kubernetes/website/issues/)
  for issues that make good entry points.
```